source/_integrations/ecowitt.markdown

```diff
@@ -22,13 +22,17 @@ ha_integration_type: integration
 
 ## Ecowitt Weather Station configuration
 
-The following steps must be performed to set up this integration. For security reason, use the token path that you get from the Home Assistant config flow.
+The Ecowitt integration works by first creating a callback endpoint on your Home Assistant instance, then adding this configuration to the Ecowitt console so that it starts sending data.
```

Comment:
```suggestion
The **Ecowitt** {% term integration %} works by first creating a callback endpoint on your Home Assistant instance and then adding this configuration to the Ecowitt console so that it starts sending data.
```